README.md

```diff
@@ -1,3 +1,52 @@
+### Onboarding sequence
+
+These are all the various components that interact during a user's onboarding.
+
+  ```mermaid
+sequenceDiagram
```

Comment:
@charleslavon let me know if this looks right and if there's any further cases in these critical paths we can handle

Reply:
@RyanSept Thanks for starting this. Updated docs will be especially helpful soon when we onboard security teams to eval our work.  FYI, I pushed an update